common/djangoapps/student/cookies.py

```diff
@@ -107,7 +112,7 @@ def set_logged_in_cookies(request, response, user):
     user_info_cookie_is_secure = request.is_secure()
 
     response.set_cookie(
-        settings.EDXMKTG_USER_INFO_COOKIE_NAME,
+        settings.EDXMKTG_USER_INFO_COOKIE_NAME.encode('utf-8'),
```

Comment:
Looking quickly through this [RFC](http://tools.ietf.org/html/rfc6265), I can see how non-ASCII characters in the cookie name could cause problems downstream (although I'm not sure if that's what you mean by "loss of information").

An alternative would be typecast the value to a `str` and catch/log any UnicodeEncode errors that might occur.